Example/SquareFlowLayoutExample.xcodeproj/xcuserdata/vladislavprusakov.xcuserdatad/xcschemes/xcschememanagement.plist

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
It's `xcuserdata` file, so it is relative only for you. It should not be included into project